src/reader/html.clj

```diff
@@ -12,21 +12,48 @@
   [html]
   (h/as-hickory (h/parse html)))
 
-(defn- remove-node-images
+(defn remove-node-images
   [node]
   (if (and (map? node)
            (or (= (:tag node) :img)
                (= (:tag node) :svg)))
     ""
     node))
 
-(defn- remove-images
-  [hickory]
-  (w/postwalk remove-node-images hickory))
+(defn remove-node-css
+  [node]
+  (if (and (map? node)
+           (or (contains? node :style)
+               (= (:tag node) :style)
+               (= (:tag node) :link)))
```

Comment:
Check the `rel` attribute on links, to make sure that they are in fact linking to a stylesheet.